modules/solid_properties/test/tests/silicon_carbide/test_with_derivatives2.i

```diff
@@ -0,0 +1,115 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [u]
+    initial_condition = 1000.0
+  []
+[]
+
+[AuxVariables]
+  [T]
+    initial_condition = 923.15
+  []
+  [cp]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [k]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [rho]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [dcp_dT]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [dk_dT]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [drho_dT]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+[]
+
+[AuxKernels]
+  [cp]
+    type = MaterialRealAux
+    variable = cp
+    property = cp_solid
+  []
+  [k]
+    type = MaterialRealAux
+    variable = k
+    property = k_solid
+  []
+  [rho]
+    type = MaterialRealAux
+    variable = rho
+    property = rho_solid
+  []
+  [dcp_dT]
+    type = MaterialRealAux
+    variable = dcp_dT
+    property = dcp_solid/dT
+  []
+  [dk_dT]
+    type = MaterialRealAux
+    variable = dk_dT
+    property = dk_solid/dT
+  []
+  [drho_dT]
+    type = MaterialRealAux
+    variable = drho_dT
+    property = drho_solid/dT
+  []
+[]
+
+[Materials]
+  [sp_mat]
+    type = ThermalSiliconCarbideProperties
+    temperature = T
+    thermal_conductivity_model = parfume
+    density = 3180.0
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+[]
+
+[BCs]
+  [left]
+    type = DirichletBC
+    variable = u
+    boundary = 'left'
+    value = 1000.0
+  []
+  [right]
+    type = DirichletBC
+    variable = u
+    boundary = 'right'
+    value = 500.0
```

Comment:
Oh I see what you mean now. There aren't any branching conditions with temperature, so I don't think it's strictly necessary to have a test that covers a particular range in temperature. If `f(x)` fails a test for _some_ `x` then it will fail for all values of `x` because there's not any branching conditions, right?

Reply:
i.e. I think the existing tests are completely sufficient checks.

Reply:
You're right that your test is sufficiently checking that future results do not differ from the results you check in. Perhaps @GiudGiud is suggesting that there be some kind of verification at some point that the correlations you're using give reasonable values over some temperature range?